### PR TITLE
Add printing out of snapshot progress

### PR DESCRIPTION
**Pull request by KrauseFx**
This way it’s easy for the user to see how many runs were already done and how many are left

This will print out the following

```
...
[19:10:12]: snapshot run 5 of 6
...
```

### CODE REVIEW

File: snapshot/lib/snapshot/runner.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class Runner
     # All the errors we experience while running snapshot
     attr_accessor :collected_errors
 
+    # rubocop:disable Metrics/AbcSize
```
Comment (mfurtak) [inline snapshot/lib/snapshot/runner.rb]: What do you think about disabling this check entirely? There are a number of places we disable it locally, and it doesn't seem to be changing the way we write our code.

File: snapshot/lib/snapshot/runner.rb [hunk 2]
```diff
@@ -32,16 +33,18 @@ def work
...
-          Snapshot.config[:languages].each do |language|
+          Snapshot.config[:languages].each_with_index do |language, language_index|
             locale = nil
             if language.kind_of?(Array)
               locale = language[1]
               language = language[0]
             end
             results[device] ||= {}
 
+            UI.message("snapshot run #{device_index * Snapshot.config[:languages].count + language_index + 1} of #{Snapshot.config[:languages].count * Snapshot.config[:devices].count}")
```
Comment (mfurtak) [inline snapshot/lib/snapshot/runner.rb]: One small style suggestion would be to make local variables for these values to clean up this line.

**Comment by mfurtak** [comment]
Looks good to me with a couple of suggestions 👍

File: snapshot/lib/snapshot/runner.rb (hunk 1 above)
Comment (KrauseFx) [inline snapshot/lib/snapshot/runner.rb]: Great point, I'll go ahead and do so in a separate PR 👍

File: snapshot/lib/snapshot/runner.rb (hunk 2 above)
Comment (KrauseFx) [inline snapshot/lib/snapshot/runner.rb]: Agreed, I'll refactor this

**Comment by KrauseFx** [comment]
Alright, I updated the PR, ready for another review 👍

**Comment by mfurtak** [comment]
🚀  ship it

**Comment by KrauseFx** [comment]
Thanks for the quick reviews 👍

**Pull request merged**